src/Core/Components/List/ListComponentBase.razor

```diff
@@ -0,0 +1,55 @@
+﻿@using Microsoft.AspNetCore.Components.Rendering
+@using System.Reflection
+
+@namespace Microsoft.FluentUI.AspNetCore.Components
+@inherits FluentComponentBase
+@typeparam TOption
+
+@code
+{
+    private void RenderOptions(RenderTreeBuilder __builder)
```

Comment:
Is it possible to move this method to the Code behind (a .cs file) ?
A future Code Analysis rule will refuse the code in razor file.

Reply:
Yes, you could to that, returning a `RenderFragment` object. The single difficulty will be to create `FluentOption` by code :-)

Reply:
Will work on that for later release